### ISSUE TITLE
Back button still doesn't work

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wai-wcag-quickref#134]
To reproduce in any browser OS combination 

I've checked 
- FF, Chrome, IE, Edge on WIN10, 
- Chrome, FF, and Safari on MacOS
1. go to WCAG https://www.w3.org/TR/WCAG20/  
2. go to 1.1.1
3. click "how to meet" on right to go to quickref
4. click back button or the backspace key

Expected result: return to WCAG 2
Current result: Stays on Quickref

**Comment by yatil** [comment]
This is now fixed in the live version as well, there was an error setting the quickref up on the W3C server. Sorry for that.
